prod/windows/createFileService.bat

```diff
@@ -0,0 +1 @@
+sc create "FileServer" start= auto displayname= "FileServer" binpath= "%~dp0runfs.bat"
```

Comment:
what was the error you got?
I think the issue is binpath, it is not accepting %~dp0 . check registry and see if your binpath is correct